models/topics.js

```diff
@@ -0,0 +1,7 @@
+const db = require('../db/connection');
+
+exports.selectTopics = () => {
+    return db.query('SELECT * FROM topics')
+    .then(({rows}) => rows)
+    .catch(err => err);
```

Comment:
we don't need a catch in the model as well as the controller - any errors that occur in the model will be caught in the controller's catch, so this can be removed :)